src/locales/default.json

```diff
@@ -556,6 +556,19 @@
       "url": "https://blog.shutter.network/announcing-shutter-governance-shielded-voting-for-daos/"
     }
   },
+  "validation": {
+    "label": "Validation",
+    "title": "Select voting validation",
+    "information": "The type of validation used to determine if a user can vote. (Enforced on all future proposals)",
+    "any": {
+      "label": "Anyone can vote",
+      "description": "Anyone can vote with voting power can cast a vote."
+    },
+    "passport": {
+      "label": "Gitcoin Passport",
+      "description": "Select a minimum amount of Gitcoin Passport stamps that a user must have to be able to vote."
```

Comment:
This is a bit misleading, you dont technicaly select a minimum amount of stamp to have, but you define a weight for each stamp and a global min weight to have to pass validation.

Reply:
Yep didn't give it much thought yet. How is this? 
<img width="489" alt="image" src="https://user-images.githubusercontent.com/51686767/199387883-e82841c2-b0fd-4654-8185-d6eeabb495f0.png">